scripts/dev/release/backup_csv_images.py

```diff
@@ -0,0 +1,370 @@
+#!/usr/bin/env python3
+"""
+Script to backup digest-pinned images from a ClusterServiceVersion to quay with _openshift_<mck_version>.
+
+This script parses a ClusterServiceVersion YAML file, extracts all digest-pinned images,
+and backs them up to the same quay registry under the with the _openshift_<mck_version> tag.
+
+"""
+
+import argparse
+import logging
+import os
+import subprocess
+import sys
+from pathlib import Path
+from typing import Dict, List, Optional
+
+import yaml
+
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(levelname)s - %(message)s'
+)
+logger = logging.getLogger(__name__)
+
+def run_command(cmd: List[str], check: bool = True, dry_run: bool = False) -> Optional[subprocess.CompletedProcess]:
+    """Run a shell command and return the result.
+
+    If dry_run is True, only log the command and return None.
+    """
+    logger.info(f"[DRY RUN] Executing: {' '.join(cmd)}" if dry_run else f"Executing: {' '.join(cmd)}")
```

Comment:
let's not add [DRY RUN] if it's not a dry run

Reply:
if its not a dry_run we are running this
```
 else f"Executing: {' '.join(cmd)}")
```

or do you mean we shouldn't use the ternary operator?